flavors/standard-EXASOL-all/flavor_base/language_definitions.json

```diff
@@ -0,0 +1,36 @@
+
+{
+  "schema_version": 1,
+  "language_definitions": [
+    {
+      "protocol": "localzmq+protobuf",
+      "aliases": ["JAVA", "JAVA11"],
+      "language": "java",
+      "parameters": [],
+      "udf_client_path": {
+        "executable": "/exaudf/exaudfclient"
+      },
+      "deprecation": null
+    },
+    {
+      "protocol": "localzmq+protobuf",
+      "aliases": ["PYTHON3", "PYTHON310"],
+      "language": "python",
```

Comment:
Is language optional? Reason is, of we have in future one binary per language the parameter would be needed anymore

Reply:
Language is not optional, see https://github.com/exasol/script-languages-container-tool/blob/1763f95ab001930eae1a99789dee0498aded91f7/exasol/slc/models/language_definition_model.py#L23